src/prerenderContent/prerenderObj.tsx

```diff
@@ -0,0 +1,63 @@
+import * as ReactDOMServer from 'react-dom/server'
+import { HelmetProvider } from 'react-helmet-async'
+import { App, helmetContext } from '../App'
+import { contentHash } from './contentHash'
+import { filenameFromUrl } from './filenameFromUrl'
+import { generateHtml } from './generateHtml'
+import { generatePreloadDump } from './generatePreloadDump'
+import { Obj, renderPage, urlFor } from 'scrivito'
+
+export async function prerenderObj(
+  obj: Obj,
+  baseHtmlTemplate: string,
+): Promise<{ filename: string; content: string }[]> {
+  HelmetProvider.canUseDOM = false
+
+  const { result, preloadDump } = await renderPage(obj, () => {
+    const bodyContent = ReactDOMServer.renderToString(<App />)
+    const { helmet } = helmetContext
+
+    return {
+      objId: obj.id(),
+      objUrl: urlFor(obj),
+      htmlAttributes: helmet?.htmlAttributes.toString() || '',
+      headContent: `
+          ${helmet?.title.toString()}
+          ${helmet?.meta.toString()}
+          ${helmet?.link.toString()}
+        `,
```

Comment:
Please adjust the spacing, so the prerendered html looks nicer.
```suggestion
      headContent: `
    ${helmet?.title.toString()}
    ${helmet?.meta.toString()}
    ${helmet?.link.toString()}
`,
```